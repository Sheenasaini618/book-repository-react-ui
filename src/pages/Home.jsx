import Navbar from "../components/Navbar";
import SearchInventory from "../components/SearchInventory";
import React from "react";

function Home() {

  return (
    <div>
        <Navbar />
        <SearchInventory/>
    </div>
  );
}

export default Home;
