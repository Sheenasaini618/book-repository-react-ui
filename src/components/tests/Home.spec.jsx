import React from 'react';
import '@testing-library/jest-dom';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Home from '../../pages/Home'

configure({ adapter: new Adapter() });

describe('<Home />', () => {

    let component;

    beforeEach(() => {

        component = shallow(<Home />);

    })

    it('should match snapshot', () => {
        expect(component.debug).toMatchSnapshot();
    });
})