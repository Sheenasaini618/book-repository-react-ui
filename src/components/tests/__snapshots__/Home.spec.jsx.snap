// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match snapshot 1`] = `[Function]`;

exports[`<Navbar /> should match snapshot 1`] = `[Function]`;
